packages/libs/eda/src/lib/core/components/computations/plugins/differentialabundance.tsx

```diff
@@ -164,7 +170,15 @@ function DifferentialAbundanceConfigDescriptionComponent({
 // Include available methods in this array.
 // 10/10/23 - decided to only release Maaslin for the first roll-out. DESeq is still available
 // and we're poised to release it in the future.
-const DIFFERENTIAL_ABUNDANCE_METHODS = ['Maaslin']; // + 'DESeq' in the future
+type DifferentialAbundanceMethodCitations = { Maaslin: string };
```

Comment:
This could be `{ Maaslin: ReactNode }`

Though be aware that `undefined` is part of this type.